<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProgActivite</class>
 <widget class="QDialog" name="ProgActivite">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>458</width>
    <height>482</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Programmation Activité</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>9</y>
     <width>441</width>
     <height>461</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>59</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>70</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Calibri</family>
        <pointsize>18</pointsize>
        <weight>75</weight>
        <bold>true</bold>
        <underline>false</underline>
       </font>
      </property>
      <property name="text">
       <string>Programmation d'une activité</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_7">
      <item>
       <widget class="QLabel" name="labelType_2">
        <property name="minimumSize">
         <size>
          <width>63</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>Nom :</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEditNom"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="labelType">
        <property name="minimumSize">
         <size>
          <width>63</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>Type : </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEditType"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="labelDescription">
        <property name="minimumSize">
         <size>
          <width>63</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>Description : </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEditDescription"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="labelLieu">
        <property name="minimumSize">
         <size>
          <width>63</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>Lieu : </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEditLieu"/>
      </item>
     </layout>
    </item>
    <item>
     <widget class="Line" name="line">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="minimumSize">
         <size>
          <width>63</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>Date et heure :</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDateTimeEdit" name="dateTimeEdit">
        <property name="calendarPopup">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="minimumSize">
         <size>
          <width>63</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>Durée : </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTimeEdit" name="timeEdit"/>
      </item>
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonValider">
        <property name="text">
         <string>Programmer</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonAnnuler">
        <property name="text">
         <string>Retour</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
